src/coreclr/src/jit/hwintrinsiclistarm64.h

```diff
@@ -12,407 +13,438 @@
-HARDWARE_INTRINSIC(AdvSimd,       SubtractWideningLower,                                       8,      2,     {INS_ssubl,          INS_usubl,          INS_ssubl,          INS_usubl,          INS_ssubl,          INS_usubl,          INS_ssubw,          INS_usubw,          INS_invalid,        INS_invalid},           HW_Category_SimpleSIMD,   HW_Flag_BaseTypeFromSecondArg|HW_Flag_SpecialCodeGen)
-HARDWARE_INTRINSIC(AdvSimd,       SubtractWideningUpper,                                      16,      2,     {INS_ssubl2,         INS_usubl2,         INS_ssubl2,         INS_usubl2,         INS_ssubl2,         INS_usubl2,         INS_ssubw2,         INS_usubw2,         INS_invalid,        INS_invalid},           HW_Category_SimpleSIMD,   HW_Flag_BaseTypeFromSecondArg|HW_Flag_SpecialCodeGen)
-HARDWARE_INTRINSIC(AdvSimd,       VectorTableLookup,                                           8,      2,     {INS_tbl,            INS_tbl,            INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},           HW_Category_SimpleSIMD,   HW_Flag_NoFlag)
-HARDWARE_INTRINSIC(AdvSimd,       VectorTableLookupExtension,                                  8,      3,     {INS_tbx,            INS_tbx,            INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},           HW_Category_SimpleSIMD,   HW_Flag_HasRMWSemantics)
-HARDWARE_INTRINSIC(AdvSimd,       Xor,                                                        -1,      2,     {INS_eor,            INS_eor,            INS_eor,            INS_eor,            INS_eor,            INS_eor,            INS_eor,            INS_eor,            INS_eor,            INS_eor},               HW_Category_SimpleSIMD,   HW_Flag_Commutative)
-HARDWARE_INTRINSIC(AdvSimd,       ZeroExtendWideningLower,                                     8,      1,     {INS_uxtl,           INS_uxtl,           INS_uxtl,           INS_uxtl,           INS_uxtl,           INS_uxtl,           INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},           HW_Category_SimpleSIMD,   HW_Flag_BaseTypeFromFirstArg)
-HARDWARE_INTRINSIC(AdvSimd,       ZeroExtendWideningUpper,                                     16,     1,     {INS_uxtl2,          INS_uxtl2,          INS_uxtl2,          INS_uxtl2,          INS_uxtl2,          INS_uxtl2,          INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},           HW_Category_SimpleSIMD,   HW_Flag_BaseTypeFromFirstArg)
+HARDWARE_INTRINSIC(AdvSimd,       Abs,                                                        -1,      1,     {INS_abs,            INS_invalid,        INS_abs,            INS_invalid,        INS_abs,            INS_invalid,        INS_invalid,        INS_invalid,        INS_fabs,           INS_invalid},     HW_Category_SIMD,                  HW_Flag_BaseTypeFromFirstArg)
```

Comment:
Git stops reporting changes for this file after this line. Is it largely just whitespace and flags/category changes to match the previous?

Reply:
It's unfortunate, so I can only suggest to review in commit-by-commit fashion.
Most of the changes come from [this commit](https://github.com/dotnet/runtime/pull/36916/commits/ddace96d30618bbd3200ab10456c41aa1919a2aa?diff=split&w=1) so to answer your question - yes, this is to account  for flags/category change. I also sorted the values in flags column in alphabetic order.

Reply:
It also seems that you added `HW_Flag_NoFloatingPointUsed` in some places. I confess that I probably haven't carefully scrutinized the changes in this file.

Reply:
> It also seems that you added HW_Flag_NoFloatingPointUsed in some places. I confess that I probably haven't carefully scrutinized the changes in this file.

Yes, I added this flag to the intrinsics that operate on general-purpose registers (e.g. crc32h, rbit, clz)